basic/07b_robot_6_dof_arm.py

```diff
@@ -80,14 +80,14 @@ def execute_trajectory(group, model, trajectory, feedback):
     exit(1)
 
 try:
-    model = hebi.robot_model.import_from_hrdf("hrdf/A-2085-06.hrdf")
+    model = hebi.robot_model.import_from_hrdf("hrdf/6-DoF_arm_example.hrdf")
```

Comment:
small thing -- probably will want to keep using the part numbers for consistency